github_connector/models/github_team_repository.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class GithubTeamRepository(models.Model):
+    _name = 'github.team.repository'
```

Comment:
I get this warning:
`github.team.repository has no _description `
so I think It's better that you add the parameter
`_description`